site/ko/guide/tensor.md

```diff
@@ -0,0 +1,303 @@
+matrixAlt = tf.reshape(matrixB, [4, 3, -1])  # 기존 내용을 4x3x5 텐서로
+                                             # 형태 변경
+
+# 형태가 변경된 텐서의 원소 개수는
+# 원래 텐서의 원소 개수와 같습니다.
+# 그러므로 다음은 원소 개수를 유지하면서
+# 마지막 차원에 사용 가능한 수가 없기 때문에 에러를 발생합니다.
+yet_another = tf.reshape(matrixAlt, [13, 2, -1])  # 에러!
+```
+
+## 자료형
+
+차원뿐만 아니라, 텐서는 자료형도 가지고 있습니다.
+전체 자료형을 확인하려면 `tf.DType`를 참고하세요.
+
+`tf.Tensor`가 한 개이상의 자료형을 가지는 것은 불가능합니다.
+임의의 데이터 구조를 직렬화한 `string`를 저장한 `tf.Tensor`는 예외입니다.
+
+`tf.cast`를 이용해서 `tf.Tensor`의 자료형을 다른 것으로 변경하는 것은 가능합니다:
+
+``` python
+# 정수형 텐서를 실수형으로 변환.
+float_tensor = tf.cast(tf.constant([1, 2, 3]), dtype=tf.float32)
+```
+
+`tf.Tensor`의 자료형을 확인하기 위해서는 `Tensor.dtype` 속성을 활용하세요.
+
+파이썬 객체를 이용해서 `tf.Tensor`를 생성할 때 자료형을 선택적으로 명시할 수 있습니다.
+그렇지 않으면 텐서플로가 데이터 표현에 적합한 자료형을 선택합니다.
+텐서플로는 파이썬 정수를 `tf.int32`로 변환하고 파이썬 실수는 `tf.float32`으로 변환합니다.
+그외에 동일한 규칙을 넘파이(numpy)를 배열로 변경할 때 사용합니다.
+
+## 텐서 계산하기(evaluate)
+
+일단 계산 그래프를 만들었다면, 특정 `tf.Tensor`를 생성하고 그것에 지정된 값을 가져오는 계산을 실행할 수 있습니다.
+이것은 대부분의 텐서플로가 작업하는데 필요할 뿐 아니라 디버깅에 종종 유용합니다.
+
+텐서를 계산하는 가장 간단한 방법은 `Tensor.eval` 메서드를 사용하는 것입니다. 예를 들어:
+
+```python
+constant = tf.constant([1, 2, 3])
+tensor = constant * constant
+print(tensor.eval())
+```
+
+`eval` 메서드는 기본 `tf.Session`이 활성화된 경우에만 작동합니다
+(자세한 내용은 [그래프와 세션](./graphs.md)에서 확인하세요).
+
+`Tensor.eval`은 텐서와 같은 내용을 가지는 넘파이 배열을 반환합니다.
+
+때때로 그 값이 이용할 수 없는 동적인 정보에 의존하기 때문에
+컨텍스트(context)가 없는 `tf.Tensor`는 계산할 수 없는 경우가 있습니다.
+예를 들어, `placeholder`인 텐서는 그 `placeholder`에 해당하는 값이 제공되지 않으면 계산할 수 없습니다.
+
+``` python
+p = tf.placeholder(tf.float32)
+t = p + 1.0
+t.eval()  # 플레이스홀더(placeholder)가 값을 가지고 있지 않기 때문에, 이것은 실패할 것입니다.
+t.eval(feed_dict={p:2.0})  # 플레이스홀더에 해당하는 값을 제공받기 때문에
+                           # 이것은 성공할 것입니다.
+```
+
+플레이스홀더뿐만 아니라 어떤 `tf.Tensor`도 값을 제공받는 것이 가능하다는 것에 유의하세요.
+
+다른 모델 구조는 `tf.Tensor`를 계산하는 것을 복잡하게 만들 수 있습니다.
+텐서플로는 함수안이나 제어 흐름안에 정의된 `tf.Tensor`를 직접 계산할 수 없습니다.
+만약에 `tf.Tensor`가 큐(queue)에 있는 값을 사용한다면,
+무언가가 큐에 들어간 후에 만 `tf.Tensor` 계산을 할 수 있습니다; 그렇지 않으면 계산은 중단될 것입니다.
+큐와 같이 작업할 때, `tf.Tensor`를 계산하기 전 `tf.train.start_queue_runners`를 호출하세요.
+
+## 텐서 출력하기
+
+디버깅을 위해서 `tf.Tensor` 값을 출력하고 싶을 것입니다.
+고급 디버깅에 대해 [tfdbg](../guide/debugger.md)에서 가이드를 제공하지만,
+텐서플로는 `tf.Tensor`값을 직접 출력할 수 있는 연산자를 가지고 있습니다.
+
+`tf.Tensor`를 출력할 때 다음과 같은 패턴을 쓰고자 하는 경우는 거의 없습니다:
+
+``` python
+t = <<some tensorflow operation>>
+print(t)  # 이것은 그래프가 생성되어질 때 기호화된 텐서(symbolic tensor)를 출력할 것입니다.
+          # 이 텐서는 이 컨텍스트(context)안에서 값을 가지고 있지 않습니다.
+```
+
+이 코드는 `tf.Tensor`의 값이 아닌 객체(지연 계산으로 표현)를 출력합니다.
+실제로 텐서플로는 두번째 인수로 전달된 `tf.Tensor` 집합을 출력하는 동안
+변경되지 않는 첫번째 텐서 인수를 반환하는 `tf.Print` 연산을 제공합니다.
+
+`tf.Print`을 제대로 사용하기 위해서는 반환된 값을 사용해야 합니다. 아래 예를 보면
+
+``` python
+t = <<some tensorflow operation>>
+tf.Print(t, [t])  # 어떤 일도 하지 않습니다
+t = tf.Print(t, [t])  # 여기서 tf.Print에 의해 반환된 값을 사용할 수 있습니다.
+result = t + 1  # 이제 결과를 계산할 때 `t` 값이 출력될 것입니다.
+```
+
+`result`를 계산할 때 이와 관련된 모든 것이 계산될 것입니다.
+`result`는 `t`와 의존성이 있고, `t`를 계산하는 것이 그 입력(`t`의 이전 값)을 출력하는 부가 효과가 있기 때문에 `t`는 출력됩니다.
```

Comment:
tf.Print is deprecated in TF 1.5 and not included in TF 2.
I suggest to remove this section.

@lamberta , What do you think about this section? :)

Reply:
If it is important to keep consistency with the original document(en), it is better not to change.

Reply:
Yes, just waiting @lamberta's comment. :)

Reply:
@wckim, I talked on this issue with billy.
Please remove this section.
I'll make a pr to english version. :)
Thanks

Reply:
Thanks, removed this section.